lib/travis/hub/config.rb

```diff
@@ -38,8 +38,9 @@ def jwt_key(type)
              repository:     { ssl_key: { size: 4096 } },
              queue:          'builds',
              limit:          { resets: { max: 50, after: 6 * 60 * 60 } },
-             notifications:  [],
-             auth:           { jwt_private_key: jwt_key(:private), jwt_public_key: jwt_key(:public), http_basic_auth: http_basic_auth }
+             notifications:  [ 'billing' ],
+             auth:           { jwt_private_key: jwt_key(:private), jwt_public_key: jwt_key(:public), http_basic_auth: http_basic_auth },
+             billing:        { url: ENV['BILLING_URL'] || 'http://localhost:9292', auth: ENV['BILLING_AUTH'] || 't0Ps3Cr3t' }
```

Comment:
@piotrmilcarz, isn't the auth_key different in default dev env?

Reply:
Nice catch @pavel-d !
Thank you. Fixed now.